src/test/java/mock/GameRepositoryMock.java

```diff
@@ -0,0 +1,39 @@
+package mock;
+
+import org.example.model.Game;
+import org.example.repository.dao.GameRepository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GameRepositoryMock implements GameRepository {
```

Comment:
you can use mockito

Reply:
Thank you! If I understood correctly, using Mockito I don't need to create my own mock, right? I didn't use it before, but i read about it and will use it next time.